src/conduits/java/com/enderio/conduits/common/integrations/ae2/AE2InWorldConduitNodeHost.java

```diff
@@ -21,25 +21,37 @@
 public class AE2InWorldConduitNodeHost implements IInWorldGridNodeHost, IExtendedConduitData<AE2InWorldConduitNodeHost> {
 
     private final AE2ConduitType type;
-    private final IManagedGridNode mainNode;
+    @Nullable
+    private IManagedGridNode mainNode = null;
 
     final LazyOptional<AE2InWorldConduitNodeHost> selfCap = LazyOptional.of(() -> this);
 
     public AE2InWorldConduitNodeHost(AE2ConduitType type) {
         this.type = type;
+        initMainNode();
+    }
+
+    private void initMainNode() {
         mainNode = GridHelper.createManagedNode(this, new GridNodeListener())
             .setVisualRepresentation(type.getConduitItem())
             .setInWorldNode(true)
             .setTagName("conduit");
+
+        assert mainNode != null;
```

Comment:
Remove assertion

Reply:
No problem, I only put it there because IntelliJ was complaining that `mainNode` could be `null` when it obviously can't be. I just found out that moving `mainNode.setIdlePowerUsage` above the `if` also "fixes" it somehow, so that's what I've done.